static/app/views/detectors/detail.tsx

```diff
@@ -83,7 +84,9 @@ export default function DetectorDetails() {
                 <IssuesList />
               </Section>
               <Section title={t('Connected Automations')}>
-                <ConnectedAutomationsList automations={[]} />
+                <ErrorBoundary mini>
```

Comment:
what does this do?

Reply:
yeah if it breaks it only breaks part of the page instead of going up to the next error boundary which is usually at the router level